vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -277,6 +278,16 @@ func (p *vzFSProvisioner) Delete(volume *v1.PersistentVolume) error {
 	}
 
 	path := mount + "/" + options["volumePath"] + "/" + options["volumeID"]
+
+	mounts, err := vstorage.ReadMounts("/proc/mounts")
```

Comment:
A provisioner is working in a container with its own mount namespace. In addition, a ploop can be mounted on another host. This problem has to be fixed in the ploop tool.